src/index.ts

```diff
@@ -43,8 +44,10 @@ function getRootAndPrefix(node: any): FSMerger.FSMergerObject {
     let { nodeType, sourceDirectory } = broccoliNodeInfo.getNodeInfo(node);
     root = nodeType == 'source' ? sourceDirectory : node.outputPath;
   }
+  root = path.normalize(root);
   return {
-    root: path.normalize(root),
+    root,
+    absRootWithSep: path.resolve(root) + path.sep,
```

Comment:
Do we have any advantages by adding `adsRootWithSep` to this function here rather than doing this `path.resolve(root) + path.sep` where we use it?

Reply:
Yeah, that would invoke that same code many times instead of just once per root of the merged fs because the caller invokes the method that needs this information many times in the lifetime of the merged fs object.